README.md

```diff
@@ -184,6 +184,41 @@ console.log($);
 2. Exceptions:
     1. 160 column limit
 
+## AWS
+
+### DNS Names
+
+DNS Records MUST adhere to the following pattern
+```text
+(instance).aws_service.(az).(region).service.environment.namespace
+```
+1. instance SHALL be used in the case that there are multiple instances of an aws_service, e.g. seperate databases.
+2. aws_service MUST refer to the specific AWS service that is being used, the correct name of a AWS service MUST be as specified as in the Boto3 Library (https://boto3.readthedocs.io/en/latest/reference/services/index.html)
+3. az OPTIONALLY can refer to a specific Availability Zone. This may indicate a design problem in the service.
+4. region OPTIONALLY can refer to a specific region.
+3. service MUST refer to the specific service.
+6. environment MUST refer to the environment the service is deployed into.
```

Comment:
this should be standardized, i.e dev vs develop vs development, prod -> production -> live etc....